BraveRewardsUI/Tipping/TippingViewController.swift

```diff
@@ -129,8 +129,21 @@ class TippingViewController: UIViewController, UIViewControllerTransitioningDele
           self.tippingView.overviewView.socialStackView.addArrangedSubview(UIImageView(image: UIImage(frameworkResourceNamed: $0.iconName)))
         })
       }
-      
-      self.tippingView.overviewView.disclaimerView.isHidden = banner.status == .verified
+    
+      if (state.ledger.walletBalances[.anonymous] ?? 0) > 0 || (state.ledger.walletBalances[.unblindedTokens] ?? 0) > 0 {
```

Comment:
We use this in two places, maybe extract to a computed property on ledger object?